### ISSUE TITLE
Code refactor

### ISSUE THREAD

**Issue by ARUNKANDULA** [csce5222/traffic_light_color_detection#30]
- Refactor existing code on [Kaggle](https://www.kaggle.com/code/thanndoduy/traffic-lights-detection-with-faster-r-cnn)
- Update the code to run for Gray scale image
- Hyperparameter setting layer (learning rate, epochs, feature vectors, color spaces, etc)

**Comment by johnsonlarryl** [comment]
Note..did not refactor code to encapsulate model training as I had many issues running Jupyter Notebooks and training model outside of the notebook.
